### ISSUE TITLE
I dont know python ... please help me

### ISSUE THREAD

**Issue by shijilt** [scottlawsonbc/audio-reactive-led-strip#62]
1. Why the "WebSocketsServer.h" is not included in the "ws2812_controller" folder ? I had to remove that line from the code to successfully upload it... I dont know if it will work...
2. How to run the .py files ?
even after typing the full path in command prompt, windows asks which program to use to open the file...

update after hours of trying....
tried to run the visualization.py file , it shows error

```
======= RESTART: C:\Users\xxx\Desktop\audio\python\visualization.py =======

Traceback (most recent call last):
  File "C:\Users\xxx\Desktop\audio\python\visualization.py", line 7, in <module>
    import microphone
  File "C:\Users\xxx\Desktop\audio\python\microphone.py", line 3, in <module>
    import pyaudio
ImportError: No module named pyaudio
>>> 
```

This project is only for experts ?

**Comment by joeybab3** [comment]
Did you do the pip installs in the instructions? You shpuld have run some commands like pip install pyaudio at spme point.

**Comment by joeybab3** [comment]
As for the websockets library, i asked about that in #60, and if you install it via library manager or just delete the line, it'll work.

**Comment by shijilt** [comment]
YES, everything were installed and I tried to run the "pip install pyaudio" again. It was already there, installed!
First I ran "conda create --name visualization-env python=3.5"
second
"activate visualization-env"
third
"conda install numpy scipy pyqtgraph"
after that
"pip install pyaudio"

all were showing success after downloading files from internet.
then I tried to run the visualization.py file using the "Idlelib"
After the opening the file I pressed "F5" then a new windows popup and shows the error!
